linalg/hypre.cpp

```diff
@@ -3878,7 +3878,7 @@ void HypreAMS::Init(ParFiniteElementSpace *edge_fespace)
       }
       else
       {
-         p = edge_fespace->GetOrder(0);
+         p = edge_fespace->GetElementDegree(0);
```

Comment:
Should there be a `MFEM_VERIFY` that the space is not of variable order?

Reply:
Added.

Reply:
Thanks, this could also be added in `HypreADS`.

Reply:
Added.